apps/cookbook/src/app/showcase/list-showcase/list-showcase.component.html

```diff
@@ -135,3 +135,7 @@ <h2>List with sections and standalone items:</h2>
     ></cookbook-iphone>
   </div>
 </div>
+
+<h2>API:</h2>
+<cookbook-api-description-properties [properties]="properties">
```

Comment:
Very nice addition with the API table, thank you! 🙏

Reply:
Anytime boss